### PR TITLE
adds first draft of bulk download docs

### PR DESCRIPTION
**Pull request by kilbergr**
This adds actual content to the documentation about Bulk Downloads.
Some is still missing--the ultimate URLs and whether we'll include instructive R code, but that can be added later.

The formatting looks weird in the html, but it shows up correctly on the site, so I recommend fetching this code and running yourself to check.

I ran prettier-fix, but it didn't pick up any changes. I tried to adhere to standards elsewhere.

### CODE REVIEW

**Comment by teovin** [comment]
Look great! One suggestion I have is to use snake case here for reporter slug and volume number since these two fields appear as `reporter_slug` and `volume_number` in VolumesMetadata.json.

File: src/templates/cap-docs-page.js [hunk 1]
```diff
@@ -52,9 +52,69 @@ export class CapDocsPage extends LitElement {
...
 					<h2 class="c-decoratedHeader" id="bulk-downloads">Bulk Downloads</h2>
+					<h3>Downloading</h3>
 					<p>
-						[We will put some documentation content about bulk downloads here.]
+						You can download data manually from the CAP website, or select URLs
+						to download programmatically.
 					</p>
+					<p>
+						To access data, identify the reporter slug and volume number, then visit
+						[URL-TO-COME]/reporter-slug/volume-number.zip. For example, to
```
Comment (teovin) [inline src/templates/cap-docs-page.js:62]: snake case here

File: src/templates/cap-docs-page.js [hunk 2]
```diff
@@ -52,9 +52,69 @@ export class CapDocsPage extends LitElement {
...
+						download the zip for Arkansas Reports (1837-2009), Volume 14, you'd visit
+						[URL-TO-COME]/ark/14.zip. You can identify the reporter slug and volume
+						number by selecting the reporter and volume from the <a href="/caselaw">
+						Caselaw</a> page and examining the URL:
+						[URL-TO-COME]/reporter=reporter-slug&volume=volume-number.
```
Comment (teovin) [inline src/templates/cap-docs-page.js:67]: snake case here

File: src/templates/cap-docs-page.js [hunk 3]
```diff
@@ -52,9 +52,69 @@ export class CapDocsPage extends LitElement {
...
+    └── 0003-01.html
+    └── etc
+						</pre>
+					</p>
+					<h3>Using Bulk Data</h3>
+					<p>
+						The .zip file can be unzipped using third-party GUI programs like
+						<a href="https://theunarchiver.com/">The Unarchiver</a> (Mac) or
+						<a href="https://www.7-zip.org/">7-zip</a> (Windows), or from the command
+						line with a command like <code>unzip volume-number.zip</code>.
```
Comment (teovin) [inline src/templates/cap-docs-page.js:102]: snake case here

**Review by mdellabitta — APPROVED** [review]

**Pull request merged**